trame_client/ui/core.py

```diff
@@ -8,12 +15,23 @@ class AbstractLayout:
     A layout needs to be bound to a server
     """
 
-    def __init__(self, _server, _root_elem, template_name="main", **kwargs):
+    def __init__(
+        self,
+        _server,
+        _root_elem,
+        template_name="main",
+        width="100%",
+        height="600px",
+        base_url="http://localhost",
```

Comment:
I can't remember exactly why at the moment, but I remember `127.0.0.1` being more robust than `localhost`

```suggestion
        base_url="http://127.0.0.1",
```

Reply:
Or this `base_url` is client-side? Hm. `localhost` *should* be fine

Reply:
Ignore this